docs/DeveloperGuide.adoc

```diff
@@ -800,18 +805,24 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |teacher with many students |find a student by name |retrieve details of a student without having to go through the entire list
```

Comment:
No need to repeat the features the current version of addressbook already has. Try to think of some features closely related to your target users.